### PR TITLE
Fix Issue related to richeditor

### PR DESCRIPTION
**Pull request by Teranode**
relation: https://github.com/octobercms/october/issues/2303

### CODE REVIEW

**Comment by LukeTowers** [comment]
@Teranode could you resolve the conflicts in the compiled asset file?

File: modules/backend/formwidgets/richeditor/assets/css/richeditor.css [hunk 1]
```diff
@@ -174,7 +174,7 @@ body.prevent-scroll.fr-mobile {position:fixed;-webkit-overflow-scrolling:touch}
 .fr-popup .fr-input-line input[type="text"]:focus,
 .fr-popup .fr-input-line textarea:focus {border-bottom:solid 2px #1e88e5;margin-bottom:0}
 .fr-popup .fr-input-line input + label,
-.fr-popup .fr-input-line textarea + label {position:absolute;top:0;left:0;font-size:12px;color:rgba(0,0,0,0);transition:color 0.2s ease 0s;-webkit-transition:color 0.2s ease 0s;-moz-transition:color 0.2s ease 0s;-ms-transition:color 0.2s ease 0s;-o-transition:color 0.2s ease 0s;z-index:3;width:100%;display:block;background:transparent;}
+.fr-popup .fr-input-line textarea + label {position:absolute;top:0;left:0;font-size:12px;color:rgba(0,0,0,0);transition:color 0.2s ease 0s;-webkit-transition:color 0.2s ease 0s;-moz-transition:color 0.2s ease 0s;-ms-transition:color 0.2s ease 0s;-o-transition:color 0.2s ease 0s;z-index:3;width:100%;display:block;background:#f9f9f9}
```
Comment (LukeTowers) [inline modules/backend/formwidgets/richeditor/assets/css/richeditor.css]: I need this to be transparent

**Comment by ayumihamsaki** [comment]
Hi I get the following results when testing:

eg

**Comment by Teranode** [comment]
i feel like some stuff changed since i submitted this PR <_<

**Comment by LukeTowers** [comment]
Probably @Teranode :) Could you fix it up?

**Comment by w20k** [comment]
`height: auto` won't work here. 100% or something like that.

**Comment by ayumihamsaki** [comment]
I've been using `height: 50vh` as a temp hack while waiting for this issue to get resolved.

**Comment by w20k** [comment]
@ayumihamsaki that's a hack for sure :D `height: 100%, or max|min-height: 100%` - but need to look deeper into the less files.

**Comment by ayumihamsaki** [comment]
`height: 100%` didn't work for me by the way.

**Comment by ayumihamsaki** [comment]
Hi, I have updated Modernizr, see my PR to allow support `vw` and `vh`, this may help this issue and pr, please see my commit here: https://github.com/octobercms/october/pull/4316/commits/5e5f19381457d5c9990863353db118753aafdde5

**Pull request closed without merging**